src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -84,7 +84,7 @@ private function init($redisClient, string $namespace, int $defaultLifetime, ?Ma
      *
      * @throws InvalidArgumentException when the DSN is invalid
      *
-     * @return \Redis|\RedisCluster|\Predis\ClientInterface According to the "class" option
+     * @return \Redis|\RedisCluster|RedisClusterProxy|RedisProxy|\Predis\ClientInterface According to the "class" option
```

Comment:
I'm not sure if this doc block should be fixed

Reply:
well, if we don't update it, the implementation would not respect the contract of the signature. So updating it is right.